.github/workflows/test-all-templates.yml

```diff
@@ -0,0 +1,63 @@
+name: Test all templates
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test-all-templates:
+    name: Test all templates
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        example: [native]
+        type: [module, view]
+        language: [java-objc, java-swift, kotlin-objc, kotlin-swift, cpp, js]
+        include:
+          - language: js
+            example: expo
+            type: module
```

Comment:
Supper minor thing but maybe we can keep one order of those keys.

Reply:
Good idea, what is your preferred order I think
type
example
language

would be nice, I will also refactor the order I give these arguments